app/src/main/res/values-nl/strings.xml

```diff
@@ -1,66 +1,66 @@
+    <string name="pin_to_the_top">Bovenaan vastzetten</string>
+    <string name="show_all">Alles weergeven</string>
     <string name="all_folders">Alle mappen</string>
     <string name="folder_view">Mapweergave</string>
     <string name="other_folder">Andere map</string>
-    <string name="show_on_map">Op kaart tonen</string>
-    <string name="unknown_location">Locatie onbekend</string>
-    <string name="no_map_application">Geen app voor kaarten gevonden</string>
-    <string name="no_camera_app_found">Geen app voor camera gevonden</string>
+    <string name="show_on_map">Weergeven op kaart</string>
+    <string name="unknown_location">Onbekende locatie</string>
+    <string name="no_map_application">Geen kaarten-app gevonden</string>
+    <string name="no_camera_app_found">Geen camera-app gevonden</string>
     <string name="increase_column_count">Meer kolommen</string>
     <string name="reduce_column_count">Minder kolommen</string>
-    <string name="change_cover_image">Afbeelding voor omslag veranderen</string>
+    <string name="change_cover_image">Omslagafbeelding wijzigen</string>
     <string name="select_photo">Foto selecteren</string>
     <string name="use_default">Standaard gebruiken</string>
     <string name="volume">Volume</string>
     <string name="brightness">Helderheid</string>
-    <string name="do_not_ask_again">Onthouden voor deze sessie</string>
-    <string name="lock_orientation">Schermrotatie vergrendelen</string>
-    <string name="unlock_orientation">Schermrotatie ontgrendelen</string>
+    <string name="do_not_ask_again">Niet opnieuw vragen (in deze sessie)</string>
+    <string name="lock_orientation">Schermoriëntatie vergrendelen</string>
+    <string name="unlock_orientation">Schermoriëntatie ontgrendelen</string>
 
     <!-- Filter -->
-    <string name="filter_media">Filter media</string>
+    <string name="filter_media">Media filteren</string>
     <string name="images">Afbeeldingen</string>
     <string name="videos">Video\'s</string>
-    <string name="gifs">GIF-bestanden</string>
-    <string name="no_media_with_filters">Geen bestanden gevonden met de huidige filters.</string>
+    <string name="gifs">GIF\'s</string>
+    <string name="no_media_with_filters">Er zijn geen bestanden gevonden met de huidige filters.</string>
     <string name="change_filters_underlined"><u>Filters aanpassen</u></string>
 
     <!-- Hide / Exclude -->
-    <string name="hide_folder_description">Deze functie verbergt de map door het bestand \'.nomedia\' toe te voegen. Alle submappen zullen ook worden verborgen. Kies \'Verborgen mappen tonen\' in de instellingen om toch verborgen mappen te kunnen inzien. Doorgaan?</string>
+    <string name="hide_folder_description">Deze functie verbergt de map door de extensie \'.nomedia\' toe te voegen. Alle submappen zullen ook worden verborgen. Kies \'Verborgen mappen weergeven\' in de instellingen om verborgen mappen alsnog te weergeven. Wilt u doorgaan?</string>
```

Comment:
No. It's not an extension, it's a file.

Reply:
.nomedia is an extension.

Reply:
.nomedia is an indicator file in Android to not index media files from that specific directory.

Reply:
Extension comes after filename. In this case .nomedia is itself a filename.

Reply:
@trubitsyn Thanks. I got that wrong then, I'll correct my translation.